config/locales/en.yml

```diff
@@ -17,6 +17,7 @@ en:
               app.js
             manifest_not_json: manifest is not proper JSON. %{errors}
             missing_manifest: Could not find manifest.json
+            symlink_in_zip: Symlinks are not allowed in the zip file
```

Comment:
https://github.com/zendesk/zendesk_apps_support/blob/master/config/locales/en.yml#L1 edit the other yml please